### ISSUE TITLE
CMD+Z does not work properly

### ISSUE THREAD

**Issue by ceyhun** [Draveness/ATProperty#4]
On Xcode 6.2 after I make an @s and it expands when I press cmd+z it doesn't remove it properly

**Comment by draveness** [comment]
Yes... But I donot consider that this is a bug. I will try to working on it in next version.

**Comment by vulgur** [comment]
cannot type `@class SomeClass;`

**Comment by draveness** [comment]
@vulgur I will fix it ASAP

**Comment by draveness** [comment]
@vulgur  fixed use @y instead
